config/secrets.yml

```diff
@@ -12,6 +12,8 @@
 
 development:
   secret_key_base: ca25c3e58abb023159f0315c3d04e0a44acd1eab0b6805f21c15379a581ebc9513621bcd6f7e133f7c16df237799263250d2533419ce882f5c90f5eb6ec86ff7
+  recaptcha_site_key: 'dev_recaptcha_site_key'
```

Comment:
problem is in production we have to update the CHEF yml with these changes. simple deployment can not update this file.